code-challenges.test.js

```diff
@@ -21,8 +21,31 @@ const people = [
 ]
 // Expected output: ["Ford Prefect is a hitchhiker.", "Zaphod Beeblebrox is president of the galaxy.", "Arthur Dent is a radio employee."]
 
+describe('arraySentence', () => {
+  it('takes in an array of objects and returns an array with a sentence about each person with their name capitalized', () => {
+    expect(arraySentence(people)).toEqual(["Ford Prefect is a hitchhiker.", "Zaphod Beeblebrox is president of the galaxy.", "Arthur Dent is a radio employee."])
+  })
+})
+
 // b) Create the function that makes the test pass.
 
+// Pseudo Code
+// Create a function named arraySentence
+// takes in an array with an object inside of it
+// map over the array 
+// access the name value in the array
+// change the letter at index 0 to uppercase
+// push that into a new array using string interpolation to add the occupation
+
+const arraySentence = (array) => {
+  nameSplit = array.map(value => value.name.split(""))
+  nameSplit.map(firstLast => firstLast.charAt(0).toUppercase + firstLast.slice(1).join(" "))
+  return firstLast + "is a" + occupation
+}
+
```

Comment:
You are close. Just look at what words you are adding to the sentence. For occupation, how do you get a value from an object?